src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1133,6 +1155,37 @@ components:
         - Native
         - ScryptoV1
 
+    ValidatorUptimeCollection:
```

Comment:
I'd say validator update endpoint should accept a collection of validator addresses we're interested in. No need for pagination.

Reply:
hmm are you sure about that requirement?

Reply:
that would mean, we need to get these addresses first, also passing 100 or more addresses doesn't sound reasonable

Reply:
The requirement is that it is easy (read - not too many queries) to get validator information and uptime for all validators (including non-active validators).

If there's a strong desire not to have it on the validator models, then we should at least make sure we can clearly explain how the validators page on the dashboard is intended to retrieve this information.

Accepting a (possibly large-ish) collection of validator addresses and letting the front end join the two queries up is reasonable, as it would also allow this endpoint to be used for a single or small number of validators.